public/theme/boltcms-2021/partials/_master.twig

```diff
@@ -0,0 +1,96 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@100..900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="{{ asset('css/app.css', 'theme') }}">
+    <link rel="stylesheet" href="https://unpkg.com/transition-style">
+    {# <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/tiny-slider/2.9.3/tiny-slider.css"> #}
+    {# <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/baguettebox.js/1.11.1/baguetteBox.min.css" #}
+    {# integrity="sha512-NVt7pmp5f+3eWRPO1h4A1gCf4opn4r5z2wS1mi7AaVcTzE9wDJ6RzMqSygjDzYHLp+mAJ2/qzXXDHar6IQwddQ==" #}
+    {# crossorigin="anonymous"/> #}
+    {# <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" #}
+    {# integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous"> #}
+    {# <script src="https://cdnjs.cloudflare.com/ajax/libs/tiny-slider/2.9.2/min/tiny-slider.js"></script> #}
+    {# <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" #}
+    {# integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" #}
+    {# crossorigin="anonymous"></script> #}
+    {# <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" #}
+    {# integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" #}
+    {# crossorigin="anonymous"></script> #}
+    {# <script src="https://cdnjs.cloudflare.com/ajax/libs/baguettebox.js/1.11.1/baguetteBox.min.js" #}
+    {# integrity="sha512-7KzSt4AJ9bLchXCRllnyYUDjfhO2IFEWSa+a5/3kPGQbr+swRTorHQfyADAhSlVHCs1bpFdB1447ZRzFyiiXsg==" #}
+    {# crossorigin="anonymous"></script> #}
+
+    {# make sure we always display a proper title: The record's title if there is one, appended with the
+           sitename. If there is no title, we append the sitename with the payoff, if there is one. #}
+    <title>
+        {%- if record|default -%}
+            {{- record|title ~ ' | ' -}}
+        {%- endif -%}
+        {{- config.get('general/sitename') -}}
+        {%- if record|default == null and config.has('general/payoff') -%}
+            {{- ' | ' ~ config.get('general/payoff') -}}
+        {%- endif -%}
+    </title>
+    {% include 'partials/_og_tags.twig' %}
+</head>
+<body>
+{% set baseurl = app.request.getSchemeAndHttpHost %}
+{% set link = canonical()|replace({ (baseurl) : ''}) %}
+{% setcontent blockquote = 'blockquote' where { url: link } returnsingle %}
+
+{% block body %}
+    {{ include('partials/_header.twig') }}
+
+    {{ widgets('main_top') }}
+
+    {% block main %}
+        {% block main_container %}
+        {% endblock main_container %}
+    {% endblock main %}
+
+    {{ widgets('main_bottom') }}
+{% endblock body %}
+
+{# {{ include('partials/_footer.twig') }} #}
+</body>
+</html>
+
+
+<script src="js/tiny-slider.js"></script>
```

Comment:
We should not have any tags after the ending `</html>`, and in this case after the end of `</body>` too :-)

Reply:
Check